src/php/ext/grpc/byte_buffer.c

```diff
@@ -16,23 +16,15 @@
  *
  */
 
-#ifdef HAVE_CONFIG_H
-#include "config.h"
+#include "byte_buffer.h"
+
+#ifdef _GNU_SOURCE
```

Comment:
Fix compile error #12360#issuecomment-326484589

Reply:
I don't particular like this solution. Can we understand the root cause better?

Reply:
done.